sample_inventories/multi_connect_workers_on_single_node.yml

```
@@ -0,0 +1,49 @@
+all:
```

```suggestion
---
all:
```

to fix yamllint errors in jenkins job

---

actually can you put a blurp at the top in the same format as the others